### PR TITLE
Create Comments controller actions, form and redesign views 

### PR DESCRIPTION
**Pull request by nikolatriki**

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  def new
+    @giveaway = Giveaway.find(params[:giveaway_id])
+    @comment = @giveaway.comments.build
+  end
+
+  def create
+    giveaway = Giveaway.find(params[:giveaway_id])
+    comment = giveaway.comments.build(comment_params)
+
+    comment.save
```
Comment (violeta-p) [inline app/controllers/comments_controller.rb:13]: I believe you need a condition here to catch behavior if comment is not saved.

**Review by violeta-p — CHANGES_REQUESTED** [review]

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (nikolatriki) [reply app/controllers/comments_controller.rb:13]: @violeta-p Yes, that is true. I will make the change. Thank you

File: app/helpers/comments_helper.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+module CommentsHelper
+end
```
Comment (violeta-p) [inline app/helpers/comments_helper.rb:4]: This helper can be removed if not used.

File: test/controllers/comments_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class CommentsControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (violeta-p) [inline test/controllers/comments_controller_test.rb:9]: Nice to have tests coverage.

Comment (nikolatriki) [reply test/controllers/comments_controller_test.rb:9]: @violeta-p I have not yet implemented Rspec.
But I will do it on the next PR, if it's not a problem?

**Review by violeta-p — APPROVED** [review]

**Pull request merged**